src/config.ts

```diff
@@ -49,7 +71,7 @@ function getConfig() {
         asyncOutputDestination: configOptions.get("sendAsyncOutputTo") as string,
         customREPLCommandSnippets: configOptions.get("customREPLCommandSnippets", []),
         customREPLCommandSnippetsGlobal: configOptions.inspect("customREPLCommandSnippets").globalValue as customREPLCommandSnippet[],
-        customREPLCommandSnippetsWorkspace: configOptions.inspect("customREPLCommandSnippets").workspaceValue as customREPLCommandSnippet[],
+        customREPLCommandSnippetsWorkspace: commands,
```

Comment:
We might want to consider existing configs.

Reply:
Those are merged earlier, but the merge logic could be extended. Currently, I'm just throwing in everything. We might merge on snippet names.